### ISSUE TITLE
An error occurs when the save data is loaded and the world is started.

### ISSUE THREAD

**Issue by nomeimei** [Roncarlos/Dimensional-Storage#1]
default.json is exists at C:\Users\xxxxxxxxxx\AppData\Roaming\Thunderstore Mod Manager\DataFolder\DysonSphereProgram\profiles\SandBox\BepInEx\config\tech

```
There was an issue loading data of mod com.jiceedev.DimensionalStorage.
Your save game could be corrupted! Please report this to mod author.

Message: Could not find a part of the path "C:\Users\xxxxxxxxxx\AppData\Roaming\Thunderstore Mod Manager\DataFolder\DysonSphereProgram\profiles\SandBox\BepInEx\plugins\Grmuche-Dimensional_Storage\config\tech\default.json"., Stacktrace:
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.Boolean anonymous, System.IO.FileOptions options) [0x0017d] in <2fa7a6a452ca43df998f07fd1486c0df>:0 
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.IO.FileOptions options, System.String msgPath, System.Boolean bFromProxy, System.Boolean useLongPath, System.Boolean checkHost) [0x00000] in <2fa7a6a452ca43df998f07fd1486c0df>:0 
  at (wrapper remoting-invoke-with-check) System.IO.FileStream..ctor(string,System.IO.FileMode,System.IO.FileAccess,System.IO.FileShare,int,System.IO.FileOptions,string,bool,bool,bool)
  at System.IO.StreamReader..ctor (System.String path, System.Text.Encoding encoding, System.Boolean detectEncodingFromByteOrderMarks, System.Int32 bufferSize, System.Boolean checkHost) [0x00067] in <2fa7a6a452ca43df998f07fd1486c0df>:0 
  at System.IO.StreamReader..ctor (System.String path, System.Text.Encoding encoding, System.Boolean detectEncodingFromByteOrderMarks, System.Int32 bufferSize) [0x00000] in <2fa7a6a452ca43df998f07fd1486c0df>:0 
  at System.IO.StreamReader..ctor (System.String path, System.Boolean detectEncodingFromByteOrderMarks) [0x0000d] in <2fa7a6a452ca43df998f07fd1486c0df>:0 
  at System.IO.StreamReader..ctor (System.String path) [0x00000] in <2fa7a6a452ca43df998f07fd1486c0df>:0 
  at (wrapper remoting-invoke-with-check) System.IO.StreamReader..ctor(string)
  at System.IO.File.ReadAllText (System.String path) [0x00000] in <2fa7a6a452ca43df998f07fd1486c0df>:0 
  at Com.JiceeDev.DimensionalStorage.Tech.TechManager.LoadConfig () [0x0002c] in <6cf7992a5cc44ab4824d2c254ee82d66>:0 
  at Com.JiceeDev.DimensionalStorage.Tech.TechManager.RegisterTechs () [0x00007] in <6cf7992a5cc44ab4824d2c254ee82d66>:0 
  at Com.JiceeDev.DimensionalStorage.DimensionalStorageMod.Init () [0x00038] in <6cf7992a5cc44ab4824d2c254ee82d66>:0 
  at Com.JiceeDev.DimensionalStorage.DimensionalStorageMod.Import (System.IO.BinaryReader reader) [0x00010] in <6cf7992a5cc44ab4824d2c254ee82d66>:0 
  at crecheng.DSPModSave.DSPModSavePlugin.CallImports (crecheng.DSPModSave.LoadOrder currentState) [0x000b3] in <f1c8c942b4b146d6b0303dd6312c1355>:0 
```

**Comment by nomeimei** [comment]
Even when placed correctly, the following error occurs

![image](https://github.com/user-attachments/assets/ea9b22b8-9c5b-47a6-801f-495af5d92b64)

**Comment by Roncarlos** [comment]
Should be fixed with 1.0.4
